src/main/scala/com/cognite/spark/datasource/FilesRelation.scala

```diff
@@ -37,8 +23,13 @@ class FilesRelation(config: RelationConfig)(val sqlContext: SQLContext)
     client.files.updateFromRead(files) *> IO.unit
   }
 
-  override def clientToResource(client: GenericClient[IO, Nothing]): Files[IO] =
-    client.files
+  override def getStreams(filters: Array[Filter])(
+      client: GenericClient[IO, Nothing],
+      limit: Option[Long],
+      numPartitions: StatusCode): Seq[fs2.Stream[IO, File]] =
```

Comment:
Why is `numPartitions` a `StatusCode`?

Reply:
Good question! I have no idea how that got there, so will use `Int` here.

Reply:
Autocomplete is a double-edged sword 😄